charts/gateway/templates/configmap.yaml

```diff
@@ -18,13 +18,16 @@ metadata:
 {{- end }}
 data:
   ACCEPT_LICENSE: {{ .Values.license.accept | quote}}
+  DISKLESS_CONFIG: {{ .Values.disklessConfig.enabled | quote }}
   SSG_CLUSTER_HOST: {{ .Values.clusterHostname }}
   SSG_JVM_HEAP: {{ .Values.config.heapSize }}
-{{- if .Values.database.enabled }}
-  {{- if .Values.database.create }}
+{{- if .Values.disklessConfig }}
```

Comment:
Check needs to be updated to `.Values.disklessConfig.enabled`

Reply:
Addressed in https://github.com/CAAPIM/apim-charts/pull/318/commits/0d5189e9571af9724fe57b2d231e71104ec93812